packages/TelegramClient-Core.package/TCCUsers.class/instance/getUserFor..st

```diff
@@ -0,0 +1,10 @@
+accessing
+getUserFor: aUserId
+
+	| promise |
+	promise := self at: aUserId ifAbsent:
+						[self at: aUserId put: Promise new.
+						self requestUserFor: aUserId.
+			 			self at: aUserId].
+	promise wait.
+	^ (promise value)
```

Comment:
Is this supposed to be indented that much?

Reply:
i tlined up with the `ifAbsent`, but one indentation should be more readable